colcon_core/shell/template/prefix_util.py.em

```diff
@@ -16,6 +16,8 @@ FORMAT_STR_SET_ENV_VAR = '@(shell_extension.FORMAT_STR_SET_ENV_VAR)'
 FORMAT_STR_USE_ENV_VAR = '@(shell_extension.FORMAT_STR_USE_ENV_VAR)'
 @{assert shell_extension.FORMAT_STR_INVOKE_SCRIPT is not None}@
 FORMAT_STR_INVOKE_SCRIPT = '@(shell_extension.FORMAT_STR_INVOKE_SCRIPT)'
+@{assert shell_extension.FORMAT_STR_INVOKE_SCRIPT is not None}@
+FORMAT_STR_CLEANUP_TRAILING_SEPARATORS = '@(shell_extension.FORMAT_STR_CLEANUP_TRAILING_SEPARATORS)'
```

Comment:
Remove assert and make this optional (no-op)

Reply:
Also naming it `FORMAT_STR_REMOVE_TRAILING_SEPARATOR` (singular) since the result command only removes one trailing separator.